### ISSUE TITLE
FVS: Use FIA2FVS to generate input for FVSOn*

### ISSUE THREAD

**Issue by lbross** [USFS-PNW/Fia-Biosum-Manager#241]
From @jsfried: We need to calculate FVS compatible site index and habitat type codes (Plant association codes) for geographies outside of PNW. One option to consider short of tossing our entire FIADB input module and reweaving BioSum with FIA2FVS (which does not seem to quite exist yet) would be to mine the latter for the info we need and are missing-- FVS compatible site class and plant association code-- these should be available in the STANDINIT FVS table they distribute.

**Comment by lbross** [comment]
From @jsfried: My sense is that this FIA2FVS effort has been very thorough and is open to input and suggestions that could accommodate BioSum specific needs, if any. Building on (by refactoring our BioSum front end) this strong foundation is going to make BioSum MUCH more supportable (e.g., because FIA and FVS will together provide the support for the FIADB to FVS input database portion of our workflow) AND move us to being a national modeling framework (since the FVS input tables are ready to go in the nationally distributed SQLite FIADB databases). 
There are a number of issues to work through with respect to plot and tree numbering and data structures. We should begin discussing these soon. Once we have the java based volume and biomass equation capability, integrating these FIA2FVS tables into our workflow is the last step needed to make BioSum a national model (and then, finally, having the potential to merit national support).

**Comment by lbross** [comment]
Need to update CreateConfigurationTextFile() because it has references to FVSIn.accdb

**Comment by sebbusby** [comment]
FVS_TREEINIT_COND.TREEVALUE is being correctly populated and the Options tab checkboxes are working properly to include or suppress associated columns in the FVSIn.db. Nice job!

**Comment by jsfried** [comment]
@lbross Still getting the DAO error when creating FVSIn databases in my grm3 project. If I delete the databases before running biosum, I can add the plots for the first state in a variant but when I go to add the 2nd state, I get the "primary key already exists") DAO error dialog. I also get the error if I don't first delete the databases and choose NO to override them.

**Comment by lbross** [comment]
Resolving this turned out to be more difficult than anticipated. BioSum does not have any functions to determine if a primary key exists in a table. I tried to write one but the API is in the ado instead of the dao, and we were using the dao exclusively in the lines of code that had the issue. Switching between the 2 APIs is problematic because the ado doesn't see the results of dao actions immediately. So ... I stopped creating the primary key(s) and the process is running fine at the same speed as before. This will be in the next .msi.
